src/global/styles/mixins/buttons.scss

```diff
@@ -15,3 +19,102 @@
   -webkit-tap-highlight-color: transparent;
   color: inherit;
 }
+
```

Comment:
I'm not sure about the design of the mixins, but let's discuss it when refactoring it anyways for the sbb-button usage :)

Reply:
I totally agree, I sense that I'll have to re-shape the whole thing anyway